go.mod

```diff
@@ -13,14 +13,23 @@ require (
 	k8s.io/utils v0.0.0-20230406110748-d93618cff8a2
 )
 
+require (
+	github.com/fsnotify/fsnotify v1.4.9 // indirect
+	github.com/go-task/slim-sprig v0.0.0-20230315185526-52ccab3ef572 // indirect
+	github.com/google/pprof v0.0.0-20210720184732-4bb14d4b1be1 // indirect
```

Comment:
This appears out of place?

Reply:
@shashank-boyapally can you please remove these unused packages?

Reply:
removed fsnotify.
slim-sprig and pprof are dependencies of ginkgo
![Screenshot from 2023-06-28 09-47-18](https://github.com/cloud-bulldozer/go-commons/assets/135713015/9669b083-1974-43c9-a15d-a39c588c43a7)